app/models/grda_warehouse/hud/client.rb

```diff
@@ -2324,6 +2325,7 @@ def split(client_ids, hmis_receiver_id, health_receiver_id, current_user)
     def merge_from(other_client, reviewed_by:, reviewed_at:, client_match_id: nil)
       raise 'only works for destination_clients' unless self.destination? # rubocop:disable Style/RedundantSelf
 
+      setup_notifier('PatientMerger') unless @notifier
```

Comment:
Why `unless @notifier`?

Reply:
`setup_notifier` sets `@notifier` so, I was only doing the setup if it hadn't been done previously.

Reply:
That's probably fine, but if we end up setting up a different notifier in some other method, it may use that instead, though unlikely given the context.